validator/src/main/java/com/amazon/aoc/helpers/RetryHelper.java

```diff
@@ -37,11 +37,16 @@ public static void retry(
       throws Exception {
     while (retryCount-- > 0) {
       try {
-        log.info("still can retry for {} times", retryCount);
+        log.info("retry attempt left : {} ", retryCount);
         retryable.execute();
         return;
       } catch (Exception ex) {
-        log.error("exception during retry, you may ignore it", ex);
+        log.info("retrying after 10 seconds");
+
+        if (retryCount == 0) {
```

Comment:
can we move this out of the loop? since when the retries exhausted, it exit the loop

Reply:
the reason I kept that inside is because we can print the actual exception after retry is exhausted (why retry failed?).